<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionMenu</class>
 <widget class="QWidget" name="SelectionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню выбора</string>
  </property>
  <widget class="QPushButton" name="btnGotoGenerate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Генерация пароля</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGotoGrade">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Оценка пароля</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyPass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Мои пароли</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
